kubernetes_cluster_autoscaler/README.md

```diff
@@ -54,7 +59,7 @@ kind: Pod
 metadata:
   name: '<POD_NAME>'
   annotations:
-    ad.datadoghq.com/controller.checks: |
+    ad.datadoghq.com/<CONTAINER_NAME>.checks: |
```

Comment:
This `<CONTAINER_NAME>` has to match the name of the container defined below line 76.
If we keep a container name hard-coded to `controller` line 76, it should be fine to keep `controller` here as well.
If we want to be able to customize the name of the container, then, we should also put `<CONTAINER_NAME>` below, line 76.

Reply:
I'll adjust line 76 to match, because the container name is set by the cluster autoscaler deployment, and could be anything.